README.md

```diff
@@ -22,3 +22,19 @@ An android app for the club so that we can connect all together to learn and exp
 * Event Proposal 
 * Dark Mode
 
+
+### Wireframes
+
+![Artboard – 1](https://user-images.githubusercontent.com/54468833/95046620-23fc5300-0702-11eb-9dc6-9d83e428d2e7.png)
```

Comment:
Two changes here - 
- Adequately size the images by providing width and height. 
- It would be nice if we can overlay these designs on a phone screen and then put it up. It'll look better.

Reply:
@SajidEqubal you can use this tag to properly customize the height. and the width of the image.
```
<img src="https://user-images.githubusercontent.com/54468833/95046620-23fc5300-0702-11eb-9dc6-9d83e428d2e7.png"
width="200" height="375">
```